resqpy/grid/create_grid_xml.py

```diff
@@ -0,0 +1,379 @@
+"""Submodule containing functions to create a grid xml file."""
+
+import logging
+
+log = logging.getLogger(__name__)
+
+import resqpy.olio.xml_et as rqet
+import resqpy.property as rprop
+from resqpy.olio.xml_namespaces import curly_namespace as ns
+
+always_write_pillar_geometry_is_defined_array = False
+always_write_cell_geometry_is_defined_array = False
+
+
+def create_grid_xml(grid,
+                    ijk,
+                    ext_uuid = None,
+                    add_as_part = True,
+                    add_relationships = True,
+                    write_active = True,
+                    write_geometry = True):
+    """Function that creates an xml file containing grid information"""
```

Comment:
Not an 'xml file' exactly: the xml is generated as a data structure in memory. The exact class of the returned item is from the standard python lxml package, I think it is: lxml.etree._Element